extensions/amp-iframe/1.0/component.js

```diff
@@ -75,7 +77,9 @@ export function Iframe({
         return;
       }
       dataRef.current = event.data;
-      attemptResize();
+      if (isIntersectingRef.current !== null) {
```

Comment:
Sorry, we don't want to call if we're intersecting. My previous code sample was incorrect, but this isn't quite correct either.
```suggestion
      if (isIntersectingRef.current === false) {
```

Reply:
This line is within `handlePostMessage` function which is called when there is a `postMessage` to resize the iframe on user action. In that case, we allow resizing the iframe even thought its in the viewport. I'm inclined to remove this check & call `attemptResize` directly and let AMP's `attemptChangeSize` or user specified callback allow/deny the resize request. WDYT? cc @caroqliu

Reply:
It depends how prescriptive we want to be about CLS compliance... Justin's recommendation would prevent document authors from having the information they need to resize while in viewport. The benefit of that is that this component will never cause shifts from resizing (great CLS compliance) but one downside is that authors can't rely on the resize callback to be called precisely when the `postMessage` is sent, which they might need for other reasons like displaying an overflow element. It does solve our problem of having two call sites (first when fired, second when exiting viewport) for one resize event.

Reply:
We do not guarantee CLS compliance (and CWV as a whole) for Bento components, right? If we only guarantee it for AMP components, then isn't it better to let the author's decide how they would like to handle resize `postMessage`?

Reply:
Bento components' value add is great CWV performance. I say let's start strict and with the option to support precise callbacks if there's a feature request?

Reply:
SGTM. Added comment and updated storybook to reflect this behavior.